harmonica/tests/test_point_gravity.py

```diff
@@ -89,237 +90,105 @@ def test_invalid_masses_array():
-            8.377501825344,
-            5.626754456652,
-            6.481205423461,
-            8.046436783272,
-            9.143676674289,
-            3.336126623489,
-            5.679435198661,
-            10.985375376975,
-            6.648460108563,
-            11.790711842140,
-            4.160678338057,
-            7.704923214409,
-            9.259465081419,
-            3.653330841953,
-            4.513884876958,
-            5.282294418322,
-            3.920614966133,
-            4.664576954878,
-        ]
+    # fmt: on
+    return (easting, northing, upward)
+
+
+@pytest.fixture(name="precomputed_potential")
+def fixture_precomputed_potential():
+    """
+    Return the potential generated by the fixture point on the fixture coords
+    """
+    # fmt: off
+    potential = GRAVITATIONAL_CONST * np.array(
+        [3.855176919424, 5.107752104793, 4.926658868677, 4.786054709408,
+         7.954613392000, 5.666420314717, 7.455942163459, 6.635464230025,
+         8.522128914471, 4.820334195373, 5.204875535728, 4.734941247442,
+         3.548003019775, 5.501387595948, 3.514542301199, 4.663049497953,
+         4.507569002018, 5.646058534931, 6.592272557173, 3.790156391155,
+         4.579153404808, 4.956798165253, 4.343342386884, 4.821298569285,
+         3.923152681360, 12.666216468561, 5.204344878310, 3.367042200379,
+         4.219521966453, 5.190941850742, 4.827149429419, 5.931663138166,
+         8.377501825344, 5.626754456652, 6.481205423461, 8.046436783272,
+         9.143676674289, 3.336126623489, 5.679435198661, 10.985375376975,
+         6.648460108563, 11.790711842140, 4.160678338057, 7.704923214409,
+         9.259465081419, 3.653330841953, 4.513884876958, 5.282294418322,
+         3.920614966133, 4.664576954878]
     )
+    # fmt: on
+    return potential
+
+
+@pytest.mark.use_numba
+def test_potential_cartesian_known_values(
+    point_mass, coordinates, precomputed_potential
+):
+    """
+    Compare the computed gravitational potential with reference values
+    """
+    point, mass = point_mass[:]
     # Compute potential gravity field on each computation point
-    results = point_mass_gravity(
-        coordinates, point_mass, mass, "potential", "cartesian"
-    )
-    npt.assert_allclose(results, reference_values)
+    results = point_gravity(coordinates, point, mass, "potential", "cartesian")
+    npt.assert_allclose(results, precomputed_potential)
+
+
+@pytest.mark.use_numba
+def test_point_mass_gravity_deprecated(point_mass, coordinates, precomputed_potential):
+    """
+    Test the soon-to-be-deprecated point_mass_gravity function
+    """
+    point, mass = point_mass[:]
+    # Check if a FutureWarning is raised
+    with warnings.catch_warnings(record=True) as warn:
+        results = point_mass_gravity(coordinates, point, mass, "potential", "cartesian")
+        assert len(warn) == 1
+        assert issubclass(warn[-1].category, FutureWarning)
+    npt.assert_allclose(results, precomputed_potential)
```

Comment:
I added a test function for the `point_mass_gravity` function that will be deprecated.